templates/static/styles.css

```diff
@@ -1,108 +1,118 @@
 body {
-    font-family: Arial, sans-serif;
-    margin: 20px;
+ font-family: Arial, sans-serif;
```

Comment:
Fix the indentation

Reply:
Sure, I'll fix the indentation issue. I'll update the PR shortly. Thank you for the feedback!

Reply:
Sure, I'll fix the indentation in the CSS file. I'll update the PR shortly. Thank you for the feedback!